test/Grpc.Net.Client.Web.Tests/GrpcWebResponseStreamTests.cs

```diff
@@ -59,6 +59,110 @@ public async Task ReadAsync_EmptyMessage_ParseMessageAndTrailers()
         Assert.AreEqual("0", trailingHeaders.GetValues("grpc-status").Single());
     }
 
+    [Test]
+    public async Task ReadAsync_HasMessage_OneByteBuffer_ParseMessageAndTrailers()
```

Comment:
Other than the internal state checks, these tests would pass without the product change, correct?